alacritty/src/display/mod.rs

```diff
@@ -969,7 +969,9 @@ impl Display {
         // Frame event should be requested before swaping buffers, since it requires surface
         // `commit`, which is done by swap buffers under the hood.
```

Comment:
```suggestion
        // Frame event should be requested before swapping buffers on Wayland, since it requires surface
        // `commit`, which is done by swap buffers under the hood.
```